webpack.common.js

```diff
@@ -18,6 +18,7 @@ const gitBranch = require('child_process')
 const config = {
     entry: {
         openmct: './openmct.js',
+        generatorWorker: './example/generator/generatorWorker.js',
```

Comment:
don't webpack the worker so we can load it from an URL

Reply:
note: i believe this change might be necessary on our closed source webpack configs. VIPER and maybe VISTA @davetsay ?

Reply:
Only if they use the generatorWorker